Lib/test/_test_multiprocessing.py

```diff
@@ -2571,6 +2571,15 @@ def raising():
 def unpickleable_result():
     return lambda: 42
 
+def waiting(args):
+    time.sleep(7)
+
+def bad_exit(value):
+    if value:
+        from sys import exit
+        exit(123)
```

Comment:
You probably want to use `os._exit` instead. `sys.exit` simply raises `SystemExit`.

Reply:
I just reproduced an existing use case reported to replicate the bug. Nonetheless, I've added one more test using `os._exit`.